.gitattributes

```diff
@@ -16,6 +16,14 @@
 # will never be auto-merged into from main. If the file doesn't exist here, then
 # any upstream changes will override ours.
 .gitattributes merge=ours
+.github/ISSUE_TEMPLATE/001_bug_report.md
+.github/ISSUE_TEMPLATE/002_enhancement.md
+.github/ISSUE_TEMPLATE/003_rfc-proposal.md
+.github/ISSUE_TEMPLATE/004_feature_request.md
+.github/ISSUE_TEMPLATE/005_hardware_support.md
+.github/ISSUE_TEMPLATE/006_nomination.md
+.github/ISSUE_TEMPLATE/007_ext-source.md
+.github/ISSUE_TEMPLATE/008_bin-blobs.md
```

Comment:
Why are you adding these back again? I see I didn't add the merge=ours to the ends of those lines. Can you do that for me here?

Reply:
Never mind. I'm doing it myself